test/system/flows/user_registration_test.rb

```diff
@@ -76,5 +97,16 @@ class UserRegistrationTest < ApplicationSystemTestCase
 
       OmniAuth.config.test_mode = false
     end
+
+    private
+    def allow_captcha_request
+      RestClient.unstub(:post)
```

Comment:
This should go away once we fully move to using webmock, but our other tests stub it right now. I didn't want to go and fix each test.